### PR TITLE
Update bitlocker-recovery-guide-plan.md

### PR DESCRIPTION
**Pull request by rafals2**
added info about accessibility tools

## Why

- Closes #[Issue Number]

### CODE REVIEW

File: windows/security/information-protection/bitlocker/bitlocker-recovery-guide-plan.md [hunk 1]
```diff
@@ -282,6 +282,13 @@ This error might occur if you updated the firmware. As a best practice, you shou
...
 
+Windows Recovery Environment (WinRE) will also ask for your BitLocker recovery key when you initiate a “Remove everything” reset from WinRE on a PC that uses "TPM+PIN" or "Password for OS drive" protector. Additionally, if BitLocker recovery is triggered on a keyboardless device with TPM-only protection, the WinRE environment, not the boot manager, will ask for the BitLocker Recovery key.  Once you've entered the key you should be able to access Win RE troubleshooting tools or start Windows normally.
+
+The BitLocker recovery screen that is shown by Windows RE has the accessibility tools like narrator and on-screen keyboard to help people enter their BitLocker recovery key. Those tools may not be available if the BitLocker recovery key is requested by the Windows boot manager.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-recovery-guide-plan.md:287]:
```suggestion
The BitLocker recovery screen that is shown by Windows RE has the accessibility tools like narrator and on-screen keyboard to help you enter your BitLocker recovery key. Those tools might not be available if the BitLocker recovery key is requested by the Windows boot manager.
```

File: windows/security/information-protection/bitlocker/bitlocker-recovery-guide-plan.md [hunk 2]
```diff
@@ -282,6 +282,13 @@ This error might occur if you updated the firmware. As a best practice, you shou
...
+
+To activate narrator during BitLocker Recovery in WinRE press Win + CTRL + ENTER
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-recovery-guide-plan.md:289]:
```suggestion
To activate narrator during BitLocker Recovery in WinRE, press Win + CTRL + ENTER
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @rafals2, ki ndly commit the minor suggested changes. Many thanks.

**Review by aczechowski — APPROVED** [review]
Thanks @rafals2 for the contribution! 😄

**Pull request merged**